cwf/gateway/registry/local_registry.go

```diff
@@ -14,6 +14,7 @@ limitations under the License.
 package registry
 
 import (
+	protos2 "magma/orc8r/lib/go/protos"
```

Comment:
Nit: I'm thinking "lib_protos" to (a) keep the refactor easy while still (b) using a descriptive name